en/docs/concepts/authentication/saml-back-channel.md

```diff
@@ -0,0 +1,50 @@
+# SAML Back-Channel Logout
+
+SAML back-channel logout enables a user to log out of an application and simulatenously log out of other connected applications without having to explicity log out of them one by one. The [SAML 2.0](TODO:link-to-concept) protocol has two approaches for single logout. 
```

Comment:
I think we can have this first section as a fragment common for both front channel and back channel. For the intro sentence, if we mention it as SAML instead of specifying it as back-channel or front-channel, the rest of it seems similar. WDYT?